unlock_indicator.c

```diff
@@ -91,6 +94,97 @@ static double scaling_factor(void) {
     return (dpi / 96.0);
 }
 
+/*
+ * Returns the cairo_font_face_t for sans-serif
+ * if this thing returns NULL, then we're somehow on a system without a
+ * sans-serif font, or a system without fontconfig (but with cairo somehow)
+ * Reference counts and caches the font-face, so we don't have memory leaks
+ * (and so that we don't have to go through all the parsing each time)
+ */
+
+static cairo_font_face_t *get_font_face() {
+    if (cairo_font_face_get_reference_count(sans_serif)) {
+        return cairo_font_face_reference(sans_serif);
+    }
+
+    FcResult result;
+    /*
+     * Loads the default config.
+     * On successive calls, does no work and just returns true.
+     */
+    if (!FcInit()) {
+        DEBUG("Fontconfig init failed. No fonts will be available...\n");
```

Comment:
What does “no fonts will be available” imply? Will no text be visible on screen? If so, make the message say that :)